Zend/zend_compile.c

```diff
@@ -5098,7 +5098,8 @@ static bool zend_compile_parent_property_hook_call(znode *result, zend_ast *ast,
 		zend_error_noreturn(E_COMPILE_ERROR, "Cannot create Closure for parent property hook call");
 	}
 
-	zend_string *property_name = zend_ast_get_str(class_ast->child[1]);
+	zval *property_hook_name_zv = zend_ast_get_zval(class_ast->child[1]);
+	zend_string *property_name = zval_get_string(property_hook_name_zv);
```

Comment:
I _think_ this will report a leak for larger numbers that don't have a `ZSTR_CHAR()` representation.

Reply:
No it won't for two reasons:
1) We actually know upfront that if we have a number we will do a fatal compile error, so no leaks here
2) Even if the previous point wouldn't be a thing, the `zval_get_string` replaces the `zend_string_copy` as well that was here previously

Reply:
Ofc, we don't show emalloc leaks on bailout, nevermind.